tests/test_isPrimaryFlag.py

```diff
@@ -161,6 +161,8 @@ def setUp(self):
 
         # Characterize the image (create PSF, etc.)
         charImConfig = CharacterizeImageConfig()
+        charImConfig.measurePsf.psfDeterminer = 'piff'
```

Comment:
Should the default psfDeterminer in `CharacterImageConfig` also be switched to "piff", instead of setting it here explicitly? In any case, it should be in double quotes for consistency with the rest of the file and in other files that have changed.

Reply:
I set it here b/c it's a unit test, and I don't want to assume anything about the API for the next great PsfEstimator.

`CharacterizeImageTask` already gets its psf estimator via `MeasurePsfTask`, which is where the default got reset.